packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -265,7 +269,6 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
     const synthDuration = await synthTimer.endAs(ioHost, 'synth');
 
     if (stackCollection.stackCount === 0) {
-      await ioHost.notify(error('This app contains no stacks', CODES.CDK_TOOLKIT_E5001));
```

Comment:
How come?

Reply:
It's a duplicate of the next line. I think in a previous PR I accidentally duplicated the message.